src/suppressUnlessProp.tsx

```diff
@@ -1,23 +1,25 @@
 import {branch, renderNothing} from 'ad-hok'
 import {some} from 'lodash/fp'
 
-import {ensureArray} from 'utils/fp'
+import {ensureArray} from './utils'
 
 type SuppressUnlessPropType = <
+  // eslint-disable-next-line @typescript-eslint/ban-types
```

Comment:
Can remove this, I turned this rule off entirely - it doesn't like the use of `{}` and recommends using `Record<string, unknown>` instead, but when I tried taking its suggestion, something didn't compile any more